lib/std/Build/Step/InstallDir.zig

```diff
@@ -21,6 +21,9 @@ pub const Options = struct {
     /// File paths which end in any of these suffixes will be excluded
     /// from being installed.
     exclude_extensions: []const []const u8 = &.{},
+    /// Only file paths which end in any of these suffixes will be included
+    /// in installation. `null` means all suffixes are valid for this option
+    include_extensions: ?[]const []const u8 = null,
```

Comment:
I think it may be worth commenting that `exclude` is prioritized over `include`. Either way is totally reasonable so I think you need documentation to make that clear (or, they can read the source).

Reply:
Would changing the second sentence to
```
`null` means all suffixes that aren't listed in `exclude_extensions` will be included in installation
```
be a better documentation version?